stan/math/prim/arr/fun/matrix_gpu.hpp

```diff
@@ -25,169 +20,195 @@ enum copy_transposed_triangular {LOWER_TO_UPPER_TRIANGULAR = 0,
 namespace stan {
   namespace math {
 
+    enum triangularity {LOWER = 0, UPPER = 1, NONE = 2 };
+    enum copy_transposed_triangular {LOWER_TO_UPPER_TRIANGULAR = 0,
+      UPPER_TO_LOWER_TRIANGULAR = 1};
+
+    /**
+     * This class represents a matrix on the GPU. 
+     * 
+     * The matrix data is stored in the oclBuffer.     
+     * 
+     */
     class matrix_gpu {
       private:
-
         cl::Buffer oclBuffer;
 
       public:
+        int rows_;
+        int cols_;
+        int rows() {
```

Comment:
These methods should be declared `const` so that they can be used on `const` arguments (this is required for a fix mentioned below to make a `matrix_gpu` argument constant).  E.g.,

```
int rows() const { return rows_; }
```

Reply:
Fixed, explained above.